worker.tf

```diff
@@ -46,6 +44,25 @@ data "ignition_systemd_unit" "worker-kubelet" {
   content = "${data.template_file.worker-kubelet.rendered}"
 }
 
+data "template_file" "worker-kubelet-conf" {
+  template = "${file("${path.module}/resources/worker-kubelet-conf.yaml")}"
+
+  vars {
+    cluster_dns   = "${var.cluster_dns}"
+    feature_gates = "${join("\n  ", formatlist("%s: %s", keys(var.feature_gates), values(var.feature_gates)))}"
```

Comment:
we discussed yesterday about a comment for the spaces/yaml format that this one keeps, as that `"\n  ` is a bit odd